spec/system/consumer/account/developer_settings_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "system_helper"
+
+describe "Developer Settings", js: true do
+  include AuthenticationHelper
+  include WebHelper
+
+  describe "as a logged in user" do
+    before do
+      login_as user
+      visit "/account"
+    end
+
+    context "when show_api_key_view is true" do
+      let(:spree_api_key) { SecureRandom.hex(24) }
+      let(:user) { create(:user, show_api_key_view: true, spree_api_key: spree_api_key) }
+
+      it "shows the developer settings tab" do
+        find("a", text: /#{I18n.t("spree.users.show.tabs.developer_settings")}/i).click
+        expect(page).to have_content I18n.t("spree.users.developer_settings.title")
```

Comment:
It would be much easier to not use I18n in specs. We are currently discussing it but I think we are settling on always using plain text.

Reply:
Yeah, I usually use plain text as well. But I copied some example code of another tab and used that style instead. :)